plugins/BEdita/API/src/Controller/ResourcesController.php

```diff
@@ -97,15 +94,126 @@ protected function findAssociation($relationship)
     {
         $relationship = Inflector::underscore($relationship);
         if (array_key_exists($relationship, $this->getConfig('allowedAssociations'))) {
-            $associations = TableRegistry::get($this->modelClass)->associations();
+            $associations = $this->Table->associations();
             foreach ($associations as $association) {
                 if ($association->property() === $relationship) {
                     return $association;
                 }
             }
         }
 
-        throw new NotFoundException(__('Relationship "{0}" does not exist', $relationship));
+        throw new NotFoundException(__d('bedita', 'Relationship "{0}" does not exist', $relationship));
+    }
+
+    /**
+     * Paginated list of resources.
```

Comment:
we can think of a small refactor later, for now:
 * change doc block in `index()` and `view()`
 * rename `view()` to something like `handle()` ?? or other name, calling `view` to *delete* seems weird :smiley: